lib/checks/aria/no-implicit-explicit-label.js

```diff
@@ -0,0 +1,10 @@
+const { aria } = axe.commons;
+
+const role = aria.getRole(node, { noImplicit: true });
+this.data(role);
+
+if (aria.isImplicitLabel(virtualNode) || aria.isExplicitLabel(node)) {
+	return undefined;
+}
+
+return false;
```

Comment:
You should use the `text.labelText` method to check for label elements. That way you don't have to create new commons, and you are using the same method that `accessibleText` has.

I also think this needs to check that the label text isn't included in the accessible text, so that something like this passes:

```html
<label>
  foo <div role="textbox" aria-label="Foo field">
</label>
```

```html
<label id="foo">
  foo <div role="textbox" aria-labelleby="foo">
</label>
```